src/tvheadend/HTSPDemuxer.cpp

```diff
@@ -164,36 +165,67 @@ void HTSPDemuxer::Abort()
   ResetStatus();
 }
 
+namespace tvheadend
+{
+
+class SubScriptionSeekTime
```

Comment:
Nitpick, but Subscription is just one word so the second "s" should not be capitalized

Reply:
Sure. Just a typo. Thanks.